.pre-commit-config.yaml

```diff
@@ -0,0 +1,13 @@
+repos:
+-   repo: https://github.com/ambv/black
+    rev: 22.3.0
+    hooks:
+    - id: black
+-   repo: https://gitlab.com/pycqa/flake8
+    rev: 3.8.4
+    hooks:
+    - id: flake8
```

Comment:
I would move flake at the end as it is a check which might be influenced by isort.

Reply:
Sure thing